product_tax_multicompany_default/models/product.py

```diff
@@ -155,13 +155,6 @@ def set_multicompany_taxes(self):
             }
         )
 
-    @api.model_create_multi
-    def create(self, vals_list):
-        new_products = super().create(vals_list)
-        for product in new_products:
-            product.set_multicompany_taxes()
-        return new_products
-
```

Comment:
@Xino61122 here's how you can fix you issues with setting the taxes on the `create`:

```suggestion
    @api.model_create_multi
    def create(self, vals_list):
        new_products = super().create(vals_list)
        for product in new_products:
            product.set_multicompany_taxes()
        new_products.invalidate_recordset(fnames=["taxes_id", "supplier_taxes_id"])
        return new_products